Example/MultipleInstances-Example/CodeExample/CodeExampleViewController.swift

```diff
@@ -0,0 +1,135 @@
+        case .bookTwo:      return .horizontal
+        }
+    }
+
+    var bookPath: String? {
+        return Bundle.main.path(forResource: self.name, ofType: "epub")
+    }
+
+    func retain(folioReaderContainer: FolioReaderContainer) {
+        let appDelegate = (UIApplication.shared.delegate as? AppDelegate)
+
+        switch self {
+        case .bookOne: appDelegate?.epubReaderOne = folioReaderContainer
+        case .bookTwo: appDelegate?.epubReaderTwo = folioReaderContainer
+        }
+    }
+}
+
+class CodeExampleViewController: UIViewController {
+
+    @IBOutlet var bookOne   : UIButton?
+    @IBOutlet var bookTwo   : UIButton?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.bookOne?.tag = Epub.bookOne.rawValue
+        self.bookTwo?.tag = Epub.bookTwo.rawValue
+
+        self.setCover(self.bookOne, index: 0)
+        self.setCover(self.bookTwo, index: 1)
+    }
+
+    private func readerConfiguration(forEpub epub: Epub) -> FolioReaderConfig {
+
+        let config = FolioReaderConfig(withIdentifier: epub.readerIdentifier)
+        config.shouldHideNavigationOnTap = epub.shouldHideNavigationOnTap
+        config.scrollDirection = epub.scrollDirection
+
+        // Custom sharing quote background
+        config.quoteCustomBackgrounds = []
+        if let image = UIImage(named: "demo-bg") {
+            let customImageQuote = QuoteImage(withImage: image, alpha: 0.6, backgroundColor: UIColor.black)
+            config.quoteCustomBackgrounds.append(customImageQuote)
+        }
+
+        let textColor = UIColor(red:0.86, green:0.73, blue:0.70, alpha:1.0)
+        let customColor = UIColor(red:0.30, green:0.26, blue:0.20, alpha:1.0)
+        let customQuote = QuoteImage(withColor: customColor, alpha: 1.0, textColor: textColor)
+        config.quoteCustomBackgrounds.append(customQuote)
+
+        return config
+    }
+
+    fileprivate func open(epub: Epub) {
+
+        guard let bookPath = epub.bookPath else {
+            return
+        }
+
+        let readerConfiguration = self.readerConfiguration(forEpub: epub)
+        let folioReaderContainer = FolioReader.presentReader(parentViewController: self, withEpubPath: bookPath, andConfig: readerConfiguration, shouldRemoveEpub: false)
+        epub.retain(folioReaderContainer: folioReaderContainer)
+    }
+    
+    private func setCover(_ button: UIButton?, index: Int) {
+        guard
+            let epub = Epub(rawValue: index),
+            let bookPath = epub.bookPath else {
+                return
+        }
+
+        do {
+            if let image = try FolioReader.getCoverImage(bookPath) {
+                button?.setBackgroundImage(image, for: .normal)
+            }
+        } catch let e as FolioReaderError {
+            print(e.localizedDescription)
+        } catch {
+            print("Unkown error")
+        }
+    }
+}
+
+// MARK: - IBAction
+
+extension CodeExampleViewController {
+
+    @IBAction func didOpen(_ sender: AnyObject) {
```

Comment:
I can't do that as we use the `tag` attribute of the parameter `sender`. The type `Any` does not have such attribute.
Building Error:
```
Value of type 'Any' has no member 'tag'
```